.gitpod.yml

```diff
@@ -1,14 +1,44 @@
+# Use the nf-core Gitpod image located at https://github.com/nf-core/tools/blob/master/nf_core/gitpod/gitpod.Dockerfile
 image: nfcore/gitpod:latest
 
+# Gitpod definition file adapted from:
+#  - https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/.gitpod.yml
+#  - https://github.com/nextflow-io/training/blob/master/.gitpod.yml
+
 tasks:
+  # TODO: Do we remove this task? Used to preview Carpentries page build
   - name: Install local page serve
     init: |
       sudo apt-get install -y ruby ruby-dev build-essential libxml2-dev
       sudo gem install bundler
-
+  # Update Nextflow if image version is behind
+  # Install tools for exercises
+  - name: Update tools for exercises
+    command: |
+      conda install --yes --update-all \
+        salmon=1.5 \
+        fastqc=0.11 \
+        multiqc=1.10 \
```

Comment:
Can you please the version for multiqc , as I tried the gitpod environment and got an error with this version of multiqc but it worked when I updated it. Thanks

Reply:
It's unpinned now.